### DISCUSSION TITLE
PDF export missing in Download menu

### DISCUSSION THREAD

**Discussion by cs-niva** [outline/outline#6264] [Self Hosting] [answered]
Hello everyone,

I am missing the PDF export in the download menu. 
I am using the latest docker image from docker.getoutline.com/outlinewiki/outline:latest
Or is PDF export only avalable in the enterprise edition?

Cheers
Chris

**Comment by tommoor** [comment]
PDF export is only available in cloud hosted as a separate PDF generation server needs to be ran

**Reply by cs-niva** [reply]
It is possible to run this PDF gereration server in the self hosted environment?

**Reply by tommoor** [reply, marked as answer]
Sorry, yes it is possible with the licensed edition as the code to connect and manage the server does not exist in community edition.

**Reply by Tasztalos69** [reply]
Are there any plans on supporting this? The missing PDF export is the only pain point for me using outline

**Reply by tommoor** [reply]
Answer is the same as above, this functionality is supported in the licensed editions.

**Reply by divStar** [reply]
Okay. I should definitely NOT consider Outline as a locally run service and NOT get used to it and therefore NOT recommend using it over Confluence in the company, in which I work as a software architect and therefore have some say in what's being used for documentation, because Outline not better than Confluence, it's just not far enough in the business sense to screw its users over (yet).

Thought about installing it locally just because of the PDF export (my wife is better at reading PDFs than browsing some Confluence-like service in our intranet I built at home) and now I see this feature is behind a paywall. Okay, it's your right to do that, but it goes to show what Outline and your company really are.

**Reply by tommoor** [reply]
I have spent 6 years building this software and giving 99% of it away for free, I'm sorry that wasn't enough for you.

**Reply by divStar** [reply]
I am sorry, but it would have been an entirely different thing if it didn't work at all or if you had been open to a PR implementing this. Sorry for having insulted you and thanks for making it available at all, because even though I don't have a use for it without the PDF feature, I am sure many others do.
Sorry.

**Reply by tommoor** [reply]
You can always "Print to PDF" from any browser, it works the same way

**Reply by hilsonp** [reply]
The print to PDF works indeed really well and the "template" is really neat.
From a mobile (iOS), you can also use the print and within the iOS print dialog, use the "share" icon to export as PDF.
This works well for my self-hosted family outline server.
